layouts/partials/breadcrumb.html

```diff
@@ -0,0 +1,29 @@
+{{ $isSingle := true -}}
+{{ with .Parent -}}
+  {{ $isSingle = .IsHome -}}
+{{ end -}}
+<nav aria-label="breadcrumb" class="td-breadcrumbs
+    {{- if $isSingle }} td-breadcrumbs__single {{- end }}">
+  <ol class="breadcrumb">
+    {{ .Site.Params.version }} / &nbsp; {{- template "breadcrumbnav" (dict "p1" . "p2" .) }}
```

Comment:
As far as I can tell this line is the only difference compared to the stock template.
This adds the version part here, right?

<img width="481" alt="image" src="https://user-images.githubusercontent.com/2205608/202197726-1f6cf6ac-44f3-4315-a9c5-dcb2a4e51a2a.png">

It looks nice, thanks. I think it's worth adding a comment so we remember why we maintain this, and perhaps we could contribute this upstream to docsy too.

Reply:
I’m a little wary of commenting in HTML templates as this can sometimes break things.

Unfortunately, this isn’t a general solution to the use case. It works in our situation, but would need to be a lot smarter to be a solution worthy of contributing upstream. It overrides all navigation, not just things on a site that you would want versioned.

Reply:
I have done some checking and HTML comments in the template files end up in the published HTML.

I have documented the change in the main config file, in the section on maintaining version numbers where it will be found easily.